java/Mekhmet-Efe_Adzhar/src/com/zoolatech/lecture6/tasks/_3/UserRepository.java

```diff
@@ -0,0 +1,11 @@
+package com.zoolatech.lecture6.tasks._3;
+
+record UserRepository(UserCache userCache, UserTable userTable) {
```

Comment:
Usually, records are used for data transfer objects (DTO). UserRepository and UserService don't contain any data that should be handled between layers - it's better to define them as regular classes.